include/rocksdb/file_system.h

```diff
@@ -1021,7 +1021,10 @@ class FSWritableFile {
                             IODebugContext* /*dbg*/) {
     return IOStatus::OK();
   }
-  virtual IOStatus Close(const IOOptions& options, IODebugContext* dbg) = 0;
+  virtual IOStatus Close(const IOOptions& /*options*/,
+                         IODebugContext* /*dbg*/) {
+    return IOStatus::OK();
```

Comment:
Returning `NotSupported` feels more correct/intuitive since closing does not actually happen in this default implementation. I am not sure why `Truncate()`'s default implementation returns OK, but `PositionedAppend()`'s default implementation is more typical.

Changing to `NotSupported` would mean internal clients of `Close()` need to handle `NotSupported`. I think that is fine.

Reply:
Yeah - NotSupported will be more appropriate than my suggested OK() https://github.com/facebook/rocksdb/pull/10049#discussion_r888473068. Thanks for fixing it @littlepig2013